src/app/components/project-map/nodes-menu/nodes-menu.component.ts

```diff
@@ -15,9 +19,39 @@ export class NodesMenuComponent {
 
     constructor(
         private nodeService: NodeService,
-        private toasterService: ToasterService
+        private nodesDataSource: NodesDataSource,
+        private toasterService: ToasterService,
+        private serverService: ServerService,
+        private settingsService: SettingsService,
+        private electronService: ElectronService
     ) {}
 
+    async startConsoleForAllNodes() {
+        if (this.electronService.isElectronApp) {
+            let consoleCommand = this.settingsService.get<string>('console_command');
+            if(consoleCommand === undefined) {
+                consoleCommand = `putty.exe -telnet \%h \%p -wt \"\%d\" -gns3 5 -skin 4`;
```

Comment:
I believe we should move it to shared default options.

Reply:
Fixed